Cartfile

```diff
@@ -1 +1 @@
-github "ReactiveX/RxSwift" "~> 6.0"
```

Comment:
According to the Carthage official documentation.
This syntax was wrong.

https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#example-cartfile

```
*** Fetching RxSwift
Failed to check out repository into /Users/giginet/Library/Caches/org.carthage.CarthageKit/dependencies/RxSwift: No object named "~> 6.0" exists (A shell task (/usr/bin/env git rev-parse ~>\ 6.0^{object} (launched in /Users/giginet/Library/Caches/org.carthage.CarthageKit/dependencies/RxSwift)) failed with exit code 128:
fatal: ambiguous argument '~> 6.0^{object}': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
)
```

Reply:
This change was same with #264